packages/react-components/src/atoms/Tag.tsx

```diff
@@ -31,14 +31,28 @@ const highlightStyles = css({
   backgroundColor: mint.rgb,
 });
 
+const disabledStyles = css({
+  borderColor: tin.rgb,
+  color: tin.rgb,
+});
+
 type TagProps = {
+  readonly disabled?: boolean;
```

Comment:
every time.

Reply:
Haha yeah I can see why you would put `disabled` (usually called that in the DOM; there's a shorthand for `={true}` in JSX). I think I went with `enabled` on the first components because `disabled` seems already negated and `!disabled` would be double negation sort of. Now it's just about preserving consistency :shrug: :sweat_smile: